### PR TITLE
Added footer.

### PR DESCRIPTION
**Pull request by NoJamesNo**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,6 +1,10 @@
 .App {
   color: #404040;
   text-align: center;
+  display: flex;
+  flex-direction: column;
+  justify-content:flex-end;
+  min-height:100vh;
```
Comment (nohharri) [inline src/App.css]: nit: space between justify-content and flex-end. Same for min-height

File: src/App.css [hunk 2]
```diff
@@ -14,7 +18,7 @@
   }
 }
 
-.App-header {
+.Dpp-header {
```
Comment (nohharri) [inline src/App.css:21]: This should be changed back to App.

File: src/App.css [hunk 3]
```diff
@@ -37,3 +41,4 @@
     transform: rotate(360deg);
   }
 }
+
```
Comment (nohharri) [inline src/App.css]: nit: random new line

File: src/App.js [hunk 4]
```diff
@@ -1,9 +1,13 @@
...
+import Footer from './containers/Footer/Footer';
+import Body from './containers/Body/Body';
+
+import './App.css';
 
 function App() {
   return (
     <div className="App">
-      <Header />
+      <div className="Header"><Header /></div>
```
Comment (nohharri) [inline src/App.js:10]: Why was classname header added?

File: src/App.js [hunk 5]
```diff
@@ -15,6 +19,8 @@ function App() {
           Learn
         </a> */}
       </header>
+      <Body />
```
Comment (nohharri) [inline src/App.js:22]: Why did you add a body? I dont know how this will be used

File: src/containers/Footer/Footer.js [hunk 6]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+import styles from './Footer.styles';
+
+
```
Comment (nohharri) [inline src/containers/Footer/Footer.js]: nit: extra line

File: src/containers/Footer/Footer.styles.js [hunk 7]
```diff
@@ -0,0 +1,24 @@
+import { findByLabelText } from "@testing-library/react";
+
+const FOOTER_PADDING = 10;
+const FOOTER_MARGIN = 50;
+
+const styles = {
+   footer: {
+      display:'flex',
+      height: 50,
+      color: 'black',
```
Comment (nohharri) [inline src/containers/Footer/Footer.styles.js:12]: nit: Unnecessary line

File: src/containers/Header/Header.styles.js [hunk 8]
```diff
@@ -15,7 +15,9 @@ const styles = {
       paddingBottom: HEADER_PADDING,
       textTransform: 'uppercase',
       fontWeight: 'bold',
+
```
Comment (nohharri) [inline src/containers/Header/Header.styles.js:18]: nit: extra line

**Review by nohharri — CHANGES_REQUESTED** [review]

File: src/App.css (hunk 2 above)
Comment (NoJamesNo) [reply src/App.css:21]: App header is used in the code a so I changed it back into App and removed it from App.js.

File: src/App.css (hunk 1 above)
Comment (NoJamesNo) [reply src/App.css]: done

File: src/App.css (hunk 3 above)
Comment (NoJamesNo) [reply src/App.css]: Done

File: src/App.js (hunk 4 above)
Comment (NoJamesNo) [reply src/App.js:10]: I was going to use it for the css page for the App.js, but realised its not needed as they already have their own style sheets.

File: src/App.js (hunk 5 above)
Comment (NoJamesNo) [reply src/App.js:22]: Body was going to be used as a place holder to create a gap between header and footer, but a different method was used.

File: src/containers/Footer/Footer.js (hunk 6 above)
Comment (NoJamesNo) [reply src/containers/Footer/Footer.js]: Done

File: src/containers/Header/Header.styles.js (hunk 8 above)
Comment (NoJamesNo) [reply src/containers/Header/Header.styles.js:18]: Done

File: src/containers/Footer/Footer.styles.js (hunk 7 above)
Comment (NoJamesNo) [reply src/containers/Footer/Footer.styles.js:12]: Done

**Comment by NoJamesNo** [comment]
LGTM
